templates/download/amd/tab-1.html

```diff
@@ -27,25 +27,18 @@ <h2 class="u-hide--small">
       <h3>Ubuntu Server 24.04</h3>
     </div>
     <div class="col-6 col-medium-3">
-      <p>The optimised Ubuntu Server 24.04 is beta for now, the certified version is coming soon.</p>
+      <p>The version of Ubuntu with up to 10 years of long term support, until April 2034.</p>
       <p>Works on:</p>
       <ul class="p-list">
         <li class="p-list__item is-ticked">AMD Kria&trade; KD240 Drives Starter Kit</li>
         <li class="p-list__item is-ticked">AMD Kria&trade; KR260 Robotics Starter Kit</li>
         <li class="p-list__item is-ticked">AMD Kria&trade; KV260 Vision AI Starter Kit</li>
       </ul>
       <hr class="p-rule" />
-      <div class="p-notification--information is-borderless u-no-margin--bottom">
-        <div class="p-notification__content">
-          <p class="p-notification__message">
-            This Ubuntu version is working on K26 SOMs as well.
-          </p>
-        </div>
-      </div>
       {% include "download/amd/_wiki_notification.html" %}
       <p>
         <a class="p-button--positive"
-           href="https://people.canonical.com/~platform/images/xilinx/kria-ubuntu-24.04/iot-limerick-kria-classic-server-2404-classic-24.04-x05-20241114.img.xz">Download 24.04</a>
+           href="https://people.canonical.com/~platform/images/xilinx/kria-ubuntu-24.04/iot-limerick-kria-classic-server-2404-classic-24.04-x07-20250423.img.xz">Download 24.04</a>
       </p>
       <p>
         <a href="https://xilinx.github.io/kria-apps-docs/kd240/linux_boot.html">Kria&trade; KD240 Getting Started Guide for Ubuntu</a>
```

Comment:
```suggestion
        <a href="https://xilinx.github.io/kria-apps-docs/kd240/linux_boot.html">Kria&trade; KD240 Getting Started Guide for Ubuntu&nbsp;&rsaquo;</a>
```
This is a standard across all our sites